StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1115ParameterMustFollowComma.cs

```diff
@@ -156,17 +157,18 @@ private void HandleAttribute(SyntaxNodeAnalysisContext context)
                 return;
             }
 
-            var previousLine = attribute.ArgumentList.Arguments[0].GetLocation().GetLineSpan().StartLinePosition.Line;
+            var previousLine = attribute.ArgumentList.Arguments[0].GetLocation().GetLineSpan().EndLinePosition.Line;
             for (int i = 1; i < attribute.ArgumentList.Arguments.Count; i++)
             {
                 var currentArgument = attribute.ArgumentList.Arguments[i];
-                var currentLine = currentArgument.GetLocation().GetLineSpan().StartLinePosition.Line;
+                var lineSpan = currentArgument.GetLocation().GetLineSpan();
+                var currentLine = lineSpan.StartLinePosition.Line;
                 if (currentLine - previousLine > 1)
                 {
                     context.ReportDiagnostic(Diagnostic.Create(Descriptor, currentArgument.GetLocation()));
```

Comment:
Could you cache `currentArgument.GetLocation()` into local variable above and use it here?